WordPress/twentytwentytwo
# Adds PHP and CSS linting

This PR adds an optional pre-commit hook that runs php and css linting based on WordPress coding standards.

It also fixes up a couple areas where our standards did not match.

**To test**
- Make a change to `functions.php` and `style.css`
- Test that you can still commit and push to the branch
- Then run `npm i` followed by `composer install`
- Make changes again in those files that will result in some stylistic errors
- Attempt to commit them
- Verify the scripts autofix where it can
- If it can't, fix it yourself and commit the result

Addresses the local side of #24
